integration-cli/docker_cli_run_test.go

```diff
@@ -2899,9 +2903,7 @@ func (s *DockerSuite) TestPermissionsPtsReadonlyRootfs(c *check.C) {
 	}
 }
 
-func testReadOnlyFile(c *check.C, filenames ...string) {
-	// Not applicable on Windows which does not support --read-only
-	testRequires(c, DaemonIsLinux, NotUserNamespace)
+func testReadOnlyFile(c *check.C, testPriv bool, filenames ...string) {
 	touch := "touch " + strings.Join(filenames, " ")
```

Comment:
It's only called from a function that already has that `testRequires` on Linux, so seemed reasonable to remove it. Actually felt lazy, because maybe this function is not necessary as it is only called from the one test.. don't know the history...